src/util/strencodings.cpp

```diff
@@ -143,7 +144,7 @@ std::string EncodeBase64(const std::string& str)
 
 std::vector<unsigned char> DecodeBase64(const char* p, bool* pf_invalid)
 {
-    static const int decode64_table[256] =
+    constexpr std::array<int,256> decode64_table =
```

Comment:
IMO there is no gain using `std::array` in these cases.

Reply:
FWIW, this is what the C++ Core Guidelines say about the gains of using `std::array`:s instead of C arrays in the general case:
* [`ES.27: Use std::array or stack_array for arrays on the stack`](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es27-use-stdarray-or-stack_array-for-arrays-on-the-stack)
* [`SL.con.1: Prefer using STL array or vector instead of a C array`](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es27-use-stdarray-or-stack_array-for-arrays-on-the-stack)

Adding as a friendly reference - not commenting on the merits of these specific changes :-)